marklogic-data-hub/src/test/java/com/marklogic/hub/entity/EntityServicesAlignmentTest.java

```diff
@@ -42,10 +47,85 @@ public void testDeployTDEWithNoEntities() {
     }
 
     @Test
-    public void testDeployTDE() throws Exception {
-        getDataHub().clearDatabase(HubConfig.DEFAULT_FINAL_SCHEMAS_DB_NAME);
-        getDataHub().clearDatabase(HubConfig.DEFAULT_STAGING_SCHEMAS_DB_NAME);
+    public void testDeployEntityWithoutBaseUri() {
+        String entity = "{\n" +
+            "  \"info\": {\n" +
+            "    \"title\": \"Customer\",\n" +
+            "    \"version\": \"0.0.1\",\n" +
+            "  },\n" +
+            "  \"definitions\": {\n" +
+            "    \"Customer\": {\n" +
+            "      \"properties\": {\n" +
+            "        \"customerId\": {\n" +
+            "          \"datatype\": \"integer\",\n" +
+            "        },\n" +
+            "        \"name\": {\n" +
+            "          \"datatype\": \"string\",\n" +
+            "          \"collation\": \"http://marklogic.com/collation/codepoint\"\n" +
+            "        }\n" +
+            "      }\n" +
+            "    }\n" +
+            "  }\n" +
+            "}";
+        File entityFile = getHubProject().getHubEntitiesDir().resolve("Customer.entity.json").toFile();
+        try {
+            mapper.writerWithDefaultPrettyPrinter().writeValue(entityFile, mapper.readTree(entity));
+            installUserArtifacts();
+            assertTrue(false, "Inserting entity without baseUri should fail");
```

Comment:
Can we use `assertThrows` with the Exception that we are expecting? 
Currently, the test will pass in multiple cases like network failure, failure to write the entity file, etc.

Reply:
Use "fail" instead - e.g. fail("Expected a validation error to be thrown").

Note though that a missing baseUri is fine, it defaults to example.org by design.